components/user/UserLink.js

```diff
@@ -19,7 +19,7 @@ export default function UserLink({ BASE_URL, link, username }) {
       )}`}
       target="_blank"
       rel="noopener noreferrer"
-      className="rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium"
+      className="rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium dark:hover:bg-secondary-low/40 hover:bg-secondary-low"
```

Comment:
I feel `/40` for the light mode hover looks better also - we will update these later with the full rebrand.

```suggestion
      className="rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium dark:hover:bg-secondary-low/40 hover:bg-secondary-low/40"
```

<img width="1677" alt="Screenshot 2023-06-14 at 10 44 50" src="https://github.com/EddieHubCommunity/LinkFree/assets/624760/ab674e3b-ece1-4507-8e12-9280c6a04965">